src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -100,19 +100,20 @@ export class HuiLightCard extends LitElement implements LovelaceCard {
 
         <div id="controls">
           <div id="slider">
-            ${supportsFeature(stateObj, SUPPORT_BRIGHTNESS)
-              ? html`
-                  <round-slider
-                    min="0"
-                    .value=${brightness}
-                    @value-changing=${this._dragEvent}
-                    @value-changed=${this._setBrightness}
-                  ></round-slider>
-                `
-              : ""}
+            <round-slider
+              min="0"
+              .value=${brightness}
+              @value-changing=${this._dragEvent}
+              @value-changed=${this._setBrightness}
+              style=${styleMap({
+                visibility: supportsFeature(stateObj, SUPPORT_BRIGHTNESS)
+                  ? "visible"
+                  : "hidden",
+              })}
+            ></round-slider>
             <paper-icon-button
               class="light-button ${classMap({
-                "slider-center": supportsFeature(stateObj, SUPPORT_BRIGHTNESS),
+                "slider-center": true,
```

Comment:
Move it out of the `classMap` if you want it to always be present

Reply:
Or maybe can even merge the `slider-center` CSS properties into `light-button` class?